docs/building-images.md

```diff
@@ -286,9 +318,8 @@ The following sections further describe optional directories that may be include
   * `manifests` - Contains locally provided manifests which will be applied to the cluster. Can be used separately or
     in combination with the manifests section in the definition file. All files in this directory will be parsed and
     the container images that they reference will be downloaded and served in an embedded artefact registry.
-  * `helm` - Contains locally provided ([K3s](https://docs.k3s.io/helm) / [RKE2](https://docs.rke2.io/helm)) Helm Custom Resources
-    which will be applied to the cluster as Helm charts. All files in this directory will be parsed and
-    the container images that they reference will be downloaded and served in an embedded artefact registry.
+  * `helm`
+    * `values` Contains [Helm values files](https://helm.sh/docs/chart_template_guide/values_files/). Each Helm chart that requires specified values must have its own values file.
```

Comment:
Is the second sentence technically correct? It's unlikely, but possible that a single values file would apply to multiple charts, in which case would they need to make a copy or could it be reused?

Reply:
👍 Will reword